doc/authentication-methods/client-certificate.md

```diff
@@ -28,20 +28,22 @@ A `SASL EXTERNAL` authentication method is disabled by default.
 In order to enable it, please add [`sasl_mechanisms` option](../Advanced-configuration.md#authentication) to MongooseIM config file.
 Its value must include a `cyrsasl_external` item.
 Obviously the list may be longer, if the system should support both the certificate and password based authentication.
-The `SASL EXTERNAL` authentication method requires using the digital
-certificates. If the `xmpp_addrs` fields are included in the certificate, they are always checked first.
-When no `xmpp_addrs` are specified, the `cn` (_common name_) field might be
-used, but it is optional. If there is more than one `xmpp_addrs` or both the
-list and `cn` field are empty, the client must include
-authorization entity. By default _common name_ is used, however it can be
-disabled by adding `{authenticate_with_cn, false}` tuple to the list of
-`auth_opts` in MongooseIM config file.
+
+The `SASL EXTERNAL` authentication method requires a client digital certificate.
+This digital certificate should contain `xmppAddr` field(s), which is always checked first.
+If there is more than one JID specified in `xmppAddr` fields, the client must include authorisation entity which corresponds to the one of the specified JIDs.
+
+When no `xmppAddr` is specified, the `cn` (common name) field might be used to provide client's username, but it is optional (enabled by default).
+Usage of a `cn` field can be disabled by adding `{authenticate_with_cn, false}` tuple to the list of `auth_opts` in MongooseIM config file.
```

Comment:
Usage of a `cn` field can be disabled by adding the `{authenticate_with_cn, false}` tuple to the list of `auth_opts` in the MongooseIM config file.